modules/account-lib/src/coin/hbar/coinTransferBuilder.ts

```diff
@@ -0,0 +1,144 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import * as Long from 'long';
+import * as proto from '@hashgraph/proto';
+import { ExtendableError, InvalidParameterValueError } from '@bitgo/sdk-core';
+import { Recipient } from './iface';
+import { TransferBuilder } from './transferBuilder';
+import { Transaction } from './transaction';
+import { isValidAddress, isValidAmount, stringifyAccountId } from './utils';
+import { BigNumber } from 'bignumber.js';
+
+declare class DuplicateMethodError extends ExtendableError {
+  constructor(message: string);
+}
+
+export class CoinTransferBuilder extends TransferBuilder {
```

Comment:
why do we need a `CoinTransferBuilder` and `TokenTransferBuilder`?

It feels like `TokenTransferBuilder` is just a `TransferBuilder` that requires you to set the `tokenName` property.

Reply:
Me and Noel discussed about this last week and majorly is reducing dependancy within token transfer and native transfer while keeping less duplicate methods. Other possibility is if later on we will support another transfer, then it can just extend from transferBuilder without affecting token or coin, so just for better design